src/app/lib/config/services/news.js

```diff
@@ -202,6 +202,12 @@ export const service = {
           href: 'https://www.bbc.com/contact/',
           text: 'Contact the BBC',
         },
+        {
+          href:
+            'https://www.bbc.com/usingthebbc/cookies/how-can-i-change-my-bbc-cookie-settings/',
+          text: 'AdChoices / Do Not Sell My Info',
+          lang: 'en-GB',
```

Comment:
I think this link should be on every service's config: 

> The link text should be in English on all services (with an English lang attribute)

Reply:
The link has been added to all services.